explainers/Omnibox/explainer.md

```diff
@@ -0,0 +1,217 @@
+
+# DevTools: Omnibox Commands
+
+Authors:
+ - [Erica Draud](https://github.com/herdraud), Program Manager
+
+## Status of this feature
+This feature is a proposed experiment. To track the project status, you can see the scenario here: [[Experiment] DUX Omnibox Commands](https://dev.azure.com/microsoft/Edge/_workitems/edit/29631932)
```

Comment:
This link only works for Microsoft employees. You might want to remove it.

Reply:
Done!